serialize.go

```diff
@@ -24,35 +24,24 @@ func (e *ErrInvalidTag) Error() string {
 	return fmt.Sprintln("invalid tag:", e.Format)
 }
 
-type ErrEmptyTagName struct{}
-
-func (e *ErrEmptyTagName) Error() string {
-	return "Tag name parameter is empty"
-}
-
 // Field is a struct to represents the domain about a field inner gody lib
 type Field struct {
 	Name  string
 	Value string
 	Tags  map[string]string
 }
 
-func Serialize(b interface{}) ([]Field, error) {
-	return RawSerialize(DefaultTagName, b)
-}
-
-// RawSerialize is a func to serialize/parse all content about the struct input
-func RawSerialize(tn string, b interface{}) ([]Field, error) {
```

Comment:
`Raw..` funcs give support for user that don't wanna use `validator` struct and help to the practice of reusability of code.